sale_product_set/wizard/product_set_add.py

```diff
@@ -14,13 +14,23 @@ class ProductSetAdd(models.TransientModel):
         "sale.order",
         "Sale Order",
         required=True,
-        default=lambda self: self.env.context.get("active_id"),
+        default=lambda self: self.env.context.get("active_id")
+        if self.env.context.get("active_model") == "sale.order"
+        else None,
```

Comment:
Isn't False the value to set for that?
```suggestion
        else False,
```

Reply:
it's the same